index.html

```diff
@@ -0,0 +1,70 @@
+<html>
+
+
+
+<main id="main">
+  
+    <h1 id="title"> Bombay Bicycle Club</h1>
+    <link rel="stylesheet" href="style.css">
+    <p>4 men Indie ensamble hailing from London</p>
+    <figure id="img-div">
+      <img
+        id="image"
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e3/BombayBicycleClub.jpg/1280px-BombayBicycleClub.jpg"
+        alt="Bombay Bicycle Club playing at the University of Nottingham"
+      />
+      <figcaption id="img-caption">
+        <p><strong>Bombay Bicycle Club</strong> are an English indie rock band from Crouch End, London, consisting of Jack Steadman, Jamie MacColl, 
+        Suren de Saram, and Ed Nash. They are guitar-fronted and have experimented with different genres, including folk, 
+        electronica, world music and indie rock.</p>
+        <p>The band were given the opening slot on 2006's V Festival after winning a competition. They subsequently released two 
+        EPs and their debut single "Evening/Morning". Since then, the band has released five albums including So Long, See You 
+        Tomorrow which topped the album charts in February 2014. The band has toured worldwide as a headlining act, playing 
+        North America, Australia, Europe and Asia.</p>
+        <p>In January 2016, the four members announced that they would pursue solo projects. Ed Nash recorded music under the name 
+        Toothless, and released his debut album in early 2017. Frontman Jack Steadman, under the name Mr Jukes, released his 
+        debut album in 2017. In January 2019, Bombay Bicycle Club announced their return to making new music and performing live 
+        after a three-year hiatus: "Since late last year we've been getting back into the swing of playing music together. More 
+        than anything it just felt great to be in the same room playing again. It made us realise what a good thing we have and 
+        has given us renewed energy and enthusiasm for the future".</p>
+      </figcaption>
+    </figure>
+    <section id="tribute-info">
+      <h2 id="headline">Check out their studio discography:</h2>
+      <ul>
+        <li><strong>2009</strong> - I Had the Blues But I Shook Them Loose </li>
+        <li>
+          <strong>2010</strong> - Flaws
+        </li>
+        <li>
+          <strong>2011</strong> - A Different Kind of Fix 
+        </li>
+        <li>
+          <strong>2014</strong> - So Long, See You Tomorrow 
+         </li>
+        <li>
+          <strong>2020</strong> - Everything Else Has Gone Wrong 
+        </li>
```

Comment:
Please avoid using inline styling if you can and instead use the styles.css file to add styling to your html files, this practice help clean up your code and making changes to your code easier and more efficient.